README.md

```diff
@@ -39,11 +53,11 @@ To uninstall, just run:
 sudo ./uninstall.sh
 ```
 
-**Troubleshooting**
+## Troubleshooting
 
 To activate logger, do in a console:
 ```
-LOG=DEBUG sudo -E ./asus_stylus.py
+sudo LOG=DEBUG asus-stylus
```

Comment:
@SelfRef It does work? `sudo LOG=DEBUG`

Reply:
Maybe only in `fish`? Hard to tell. Fixed.